example_project/example_project/orchestra_settings.py

```diff
@@ -40,9 +40,9 @@ def setup_orchestra(settings_module_name):
     settings.INSTALLED_APPS += (
         'compressor',
         'django_object_actions',
-        'registration',
         'orchestra',
         'beanstalk_dispatch',
+        'registration',
```

Comment:
I think my rebase just 'did the right thing', but let's make sure of that after I push these changes.